cmd/beemo/main.go

```diff
@@ -113,7 +118,7 @@ func pollNewReports(cctx *cli.Context) error {
 	xrpcc.Auth.Did = auth.Did
 	xrpcc.Auth.Handle = auth.Handle
 
-	adminToken := cctx.String("admin-token")
+	adminToken := cctx.String("admin-password")
```

Comment:
is it a password?

Reply:
Yeah, and it was confusing initially when I thought it was a token. The way it works is HTTP `Basic` auth (not `Bearer`) with username `admin` and this as the password.

We are planning to switch and use regular auth (the JWT tokens, as HTTP `Bearer` auth), plus an admin flag on accounts, but this is how it works today.

Some interfaces, like redsky, do say "token" though.